login-workflow/docs/language-support.md

```diff
@@ -8,6 +8,101 @@ This package supports translations to different languages using [i18next](https:
 -   Spanish
 -   Simplified Chinese
 
-## Changing the Language
+## Add/Override translations within our workflow
 
-<!-- @TODO: Finish this... -->
\ No newline at end of file
+To add i18n in your application you can follow the instructions from [official website](https://react.i18next.com/getting-started).
+You need to wrap your app with <I18nextProvider/> for rendering translations in the app. For reference, you can check [I18nextProvider](https://react.i18next.com/latest/i18nextprovider).
+
+
+If you want to use keys from your application dictionary in any of the workflow screens or any custom screens that you provide within the Auth or Login workflows, you need to pass your app's i18n instance to the `AuthContextProvider`/`RegistrationContextProvider` via the `i18n` prop.
+
+```tsx
+import { i18n } from './i18n';
+
+<AuthContextProvider
+    ...
+    i18n={i18n}
+>
+```
+
+`Registration workflow` works in the same way, allowing you to pass an i18n instance via the `i18n` prop.
+
+```tsx
+import { i18nAppInstance } from './i18n';
+
+<RegistrationContextProvider
+    ...
+    i18n={i18n}
+>
+```
+
+If you need to override any of the translations used internally in our Workflow, you can do it in a similar way as below.
+
+```tsx
+import { useTranslation } from 'react-i18next';
+
+const { t } = useTranslation();
+<ForgotPasswordScreen emailLabel={`${t('EMAIL')}`} />
+```
+
+## Add New language within our workflow
+
+To add new languages other than English, French, Portuguese, Spanish, and Simplified Chinese. You need to override existing translation keys. You can check the following example.
+
+To override any of the strings or translations used internally in the Workflow you can do it same as below.
+```tsx
```

Comment:
This example is unclear, we are not telling user, where exactly these changes should be done (in which file).